build.sh

```diff
@@ -41,10 +45,10 @@ _exit_error() { echo "ERROR: $*" 1>&2; exit 1; }
 _log_info() { echo "INFO: $*"; }
 
 _fetch_url() {
-	mkdir -p ${BUILD_CACHE}
+	mkdir -p "${BUILD_CACHE}"
```

Comment:
And ya per our own instructions, hella quotes were missing everywhere 👍

Reply:
Which instructions? The google shell style guide also seems to suggest pretty aggressive quoting
```
Always quote strings containing variables, command substitutions, spaces or shell meta characters, unless careful unquoted expansion is required.
Prefer quoting strings that are "words" (as opposed to command options or path names).
Never quote literal integers.
Be aware of the quoting rules for pattern matches in [[.
Use "$@" unless you have a specific reason to use $*.
```

Just want to make sure there aren't shellcheck rules you would prefer ignored. It would also be great to have shellcheck run in travis. I could add that now, or as a follow up if you prefer.

Reply:
Was just noting that this script didn't follow the google style guide on quoting - thanks for adding them all

Reply:
👍